I accidentally did this while creating a simple sample.

I'm so glad I wasn't "live coding" in front of a full house.

{code}
scala> is foreach { i => println("Hi!") ; Console println 2 * i }
java.lang.IllegalArgumentException: !") ; Console println 2 * i }: event not found
{code}
